resources/js/Pages/Profile/Partials/UpdateProviders.vue

```diff
@@ -63,29 +62,29 @@ defineProps({
       </div>
 
       <div class="mt-5 space-y-6">
-        <div v-for="provider in providers" :key="provider" class="flex items-center">
-          <img :src="`/img/auth/${provider}.svg`" :alt="provider" class="auth-provider relative" />
+        <div v-for="(provider, id) in providers" :key="id" class="flex items-center">
+          <img :src="provider.logo ?? `/img/auth/${id}.svg`" :alt="provider.name" class="auth-provider relative" />
```

Comment:
Shouldn't this come from the backend (through the view helpers)?

Reply:
ok, good idea